requirements-min.txt

```diff
@@ -2,7 +2,7 @@
 h5py==2.10  # support for selection of datasets with list of indices added in 2.10
 importlib-metadata==4.2.0; python_version < "3.8"  # TODO: remove when minimum python version is 3.8
 importlib-resources==5.12.0; python_version < "3.9"  # TODO: remove when when minimum python version is 3.9
-jsonschema==2.6.0
+jsonschema>=2.6.0
```

Comment:
The point of this file is to be used when testing whether the minimum requirements set in `pyproject.toml` are valid. Using >= instead of == defeats the point.

Reply:
If 2.6.0 is too small, then the minimum version should be increased both here and in `pyproject.toml`